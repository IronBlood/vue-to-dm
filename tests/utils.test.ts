import {
	rename_to_underline,
} from "../src/utils";

describe("test rename filename", () => {
	it("rename Foo", () => {
		expect(rename_to_underline("Foo")).toMatchSnapshot();
	});

	it("rename FooComponent", () => {
		expect(rename_to_underline("FooComponent")).toMatchSnapshot();
	});

	it("rename foo-component", () => {
		expect(rename_to_underline("foo-component")).toMatchSnapshot();
	});

	it("rename Foo-Component", () => {
		expect(rename_to_underline("Foo-Component")).toMatchSnapshot();
	});
});

