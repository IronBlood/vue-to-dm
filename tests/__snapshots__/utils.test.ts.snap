// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test rename filename rename Foo 1`] = `"foo"`;
exports[`test rename filename rename FooComponent 1`] = `"foo_component"`;
exports[`test rename filename rename foo-component 1`] = `"foo_component"`;
exports[`test rename filename rename Foo-Component 1`] = `"foo_component"`;

